frb_utils/lib/src/dart_web_test_utils/run_test.dart

```diff
@@ -22,6 +22,13 @@ Future<void> executeTestWeb(TestWebConfig config) async {
   final webRoot = '$dartRoot/web';
   print('executeTestWeb: Pick dartRoot=$dartRoot');
 
+  List<String> cargoArgs = [];
+
+  if (dartRoot.endsWith('frb_example/pure_dart') ||
+      dartRoot.endsWith('frb_example/pure_dart_pde')) {
+    cargoArgs = ['--features', 'internal_feature_for_testing'];
+  }
+
```

Comment:
nit: not sure why I missed this... the `executeTestWeb` seems to be general-purpose, thus it is great to avoid hardcode such things here.
For example, make it an argument in TestWebConfig